.github/workflows/build-rust-cross-platform.yml

```diff
@@ -44,8 +46,18 @@ jobs:
       - name: Build Rust
         env:
           RUSTFLAGS: "-D warnings"
+        if: ${{ matrix.settings.target }} != aarch64-unknown-linux-gnu
         run: cargo build --target ${{ matrix.settings.target }} --release
 
+      - name: Build Rust
+        env:
+          RUSTFLAGS: "-D warnings"
+        if: ${{ matrix.settings.target }} == aarch64-unknown-linux-gnu
+        run: |
+          sudo apt update
+          sudo apt install -y crossbuild-essential-arm64
+          cargo build --target ${{ matrix.settings.target }} --release
+  
```

Comment:
Use cross for this, that's what we do in the other workflows.

Reply:
Actually, [looking at this](https://github.com/bitwarden/sdk/actions/runs/8281072317/job/22658818729#step:7:110), cross apparently doesn't have the right build tools installed? I thought `cross` pulled a container image that was configured for whatever target you specified.

Reply:
[This](https://github.com/bitwarden/sdk/pull/653/commits/be03087f123511668469417f6c104b77e2058918#) isn't running with cross? I referenced the Build CLI job to base that off of.